### ISSUE TITLE
!important rules in scss logic are showing as errors.

### ISSUE THREAD

**Issue by pixilz** [liuderchi/ide-css#13]
`Atom: 1.23.0`
`ide-css: 0.2.0`

My code is compiling and working as expected but is being marked with errors.

I have the following SCSS code:
```
/**
 * This file contains mixins that are only applicable to the styleguide.
 */

@function get-mixin-important-var($important) {
	//If important is true it will add an important tag to everything.
	@return if($important, !important, null);
}
```
The error underlines directly underneath the `!`.

IDE-CSS is reporting
![image](https://user-images.githubusercontent.com/20696274/33960248-5e2ad0ee-e007-11e7-9dac-70353106fbc6.png)

Example Usage:
```
@mixin heading($important: false) {
	$i: get-mixin-important-var($important);

	font-family: 'arial' $i;
	font-weight: 600 $i;
        color: #000 $i;

	font-size: 16px $i;
}

.heading {
      @include heading(true);
}

<div class="heading">My Heading</div>
```

Output CSS from example:
![image](https://user-images.githubusercontent.com/20696274/33960518-369a880c-e008-11e7-86dd-7e9be192fbda.png)

[Pin for example](https://codepen.io/anon/pen/OzPEGe)

**Comment by liuderchi** [comment]
@pixilz Thanks for your report.

IMO this issue is related to the css language server.
I am under a survey/experiment and will report it to correct repo.

**Comment by liuderchi** [comment]
@pixilz latest PR has been released at v0.3.0.
Please upgrade ide-css and hopefully this issue will be solved.
